homeassistant/auth/auth_store.py

```diff
@@ -235,7 +235,7 @@ def __init__(self, hass: HomeAssistant) -> None:
                 continue
             token_type = rt_dict.get('token_type')
             if token_type is None:
-                if rt_dict['clinet_id'] is None:
+                if rt_dict['client_id'] is None:
```

Comment:
Shouldn't we use constants for these strings?

Reply:
Maybe. This is only used for loading/restoring.